brave/browser-laptop
# Payment History CSV export download has correct filename (+row sorting and total row)

Payment receipt CSV links now suggest a download filename with reconciliation date in it matching UI.

~~depends on https://github.com/brave/ledger-client/pull/17/commits/d50d59e6db865a4d2da1bb4b56fe634f5f645d18~~

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist. (part of #4332)

Test Plan:
1. Open/Enable Brave Payments
2. Browse pages, wait for or trigger reconciliation
3. Click 'View Payment History' link
4. Click a receipt link, e.g. `brave_ledger9-23-2016.csv`
5. Observe that a filename is suggested in download box matching link text.
6. Download CSV
7. Open from downlloads bar
8. Observe that rows are sorted alphabetically by Publisher
9. Observe that there is now a "TOTAL" row at bottom with totals for all non-publisher columns

![brave_csv_export_filename_4332](https://cloud.githubusercontent.com/assets/6969859/18901831/1bd7167c-8503-11e6-8ad3-fbbe5aaf8fa3.gif)
